sample-app/Jenkinsfile

```diff
@@ -19,20 +19,12 @@ node {
   switch (env.BRANCH_NAME) {
     // Roll out to staging
     case "staging":
-        // Change deployed image in staging to the one we just built
-        sh("sed -i.bak 's#gcr.io/cloud-solutions-images/gceme:1.0.0#${imageTag}#' ./k8s/staging/*.yaml")
-        sh("kubectl --namespace=production apply -f k8s/services/")
```

Comment:
If these 2 lines are removed then changes to labels and deployment args will not be picked up.